### PR TITLE
Freshen up subecosystems of Moonbeam

### PR DESCRIPTION
**Pull request by themacexpert**
This PR freshens a variety of ecosystems, adding missed repos and removing stale ones. This PR does **not** modify what is considered a subecosystem of Moonbeam network.

### CODE REVIEW

File: data/ecosystems/m/moonbeans.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ title = "Moonbeans"
 
 sub_ecosystems = []
 
-github_organizations = []
+github_organizations = ["https://github.com/m00nbeans"]
```
Comment (eherrerosj) [inline data/ecosystems/m/moonbeans.toml:6]: unfortunately this is not an org link. Instead, it's a user link. Please remove it and add the repos that correspond individually.

File: data/ecosystems/m/moonwell.toml [hunk 2]
```diff
@@ -30,14 +30,11 @@ url = "https://github.com/moonwell-fi/moonwell-subgraph"
...
-
 [[repo]]
 url = "https://github.com/moonwell-fi/proposal-verification"
 
 [[repo]]
 url = "https://github.com/moonwell-fi/temporal-governance"
 
 [[repo]]
-url = "https://github.com/moonwell-fi/yield-server"
+url = "https://github.com/moonwell-fi/yield-server"
```
Comment (eherrerosj) [inline data/ecosystems/m/moonwell.toml:40]: I'm seeing some changes that are going to conflict with your other PR (https://github.com/electric-capital/crypto-ecosystems/pull/613)

File: data/ecosystems/s/seascape-network.toml [hunk 3]
```diff
@@ -30,78 +24,6 @@ url = "https://github.com/blocklords/blocklords2d-smartcontracts"
...
-url = "https://github.com/blocklords/sds-evm-adapter"
-
-[[repo]]
-url = "https://github.com/blocklords/sds-hardhat"
-
-[[repo]]
-url = "https://github.com/blocklords/sds-truffle"
-
-[[repo]]
-url = "https://github.com/blocklords/sds-ts"
```
Comment (eherrerosj) [inline data/ecosystems/s/seascape-network.toml:85]: a lot of these were reassigned to https://github.com/seascapenetwork/... org, could you re-add them with the new owner instead please?

File: data/ecosystems/s/seascape-network.toml [hunk 4]
```diff
@@ -6,12 +6,6 @@ sub_ecosystems = []
 github_organizations = ["https://github.com/blocklords"]
```
Comment (eherrerosj) [inline data/ecosystems/s/seascape-network.toml:6]: please add https://github.com/seascapenetwork/ here, it was renamed

File: data/ecosystems/t/toad-network.toml [hunk 5]
```diff
@@ -3,7 +3,7 @@ title = "Toad Network"
 
 sub_ecosystems = []
 
-github_organizations = []
+github_organizations = ["https://github.com/ToadNetwork"]
```
Comment (eherrerosj) [inline data/ecosystems/t/toad-network.toml:6]: nice

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a few comments

**Review by eherrerosj — CHANGES_REQUESTED** [review]
hey @themacexpert, you think we could add a missing tag instead of removing repos that don't exist anymore publicly? We prefer to keep non-existant repos so we know they existed at some point and we can keep their commits. 

Example: https://github.com/electric-capital/crypto-ecosystems/blob/master/data/ecosystems/h/helios-labs.toml#L11

**Comment by themacexpert** [comment]
> hey @themacexpert, you think we could add a missing tag instead of removing repos that don't exist anymore publicly? We prefer to keep non-existant repos so we know they existed at some point and we can keep their commits.
> 
> Example: https://github.com/electric-capital/crypto-ecosystems/blob/master/data/ecosystems/h/helios-labs.toml#L11

Done!

**Review by eherrerosj — APPROVED** [review]
LGTM, nice work, thx!

**Pull request merged**